lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -1847,7 +2291,7 @@ class MongoClientInterface {
             if (params.secondaryStreamParams) {
                 // eslint-disable-next-line no-param-reassign
                 newParams.mainStreamParams.gte = range[0];
-                newParams.secondaryStreamParams.gte = range[1];
+                newParams.secondaryStreamParams!.gte = range[1];
```

Comment:
same, but without any "trick" :

```suggestion
            if (newParams.secondaryStreamParams) {
                // eslint-disable-next-line no-param-reassign
                newParams.mainStreamParams.gte = range[0];
                newParams.secondaryStreamParams.gte = range[1];
```